Makefile

```diff
@@ -20,11 +20,18 @@ cljfmt:
 	lein with-profile -user,+$(VERSION),+cljfmt cljfmt check
 
 kondo:
-	clj-kondo --lint src
+	if command -v clj-kondo > /dev/null; then \
+	  clj-kondo --lint src; \
+	else \
+	  lein with-profile +kondo run -m clj-kondo.main --lint src; \
```

Comment:
Nice initiative 👍

You can go as far as eliminating the `clj-kondo` binary, see https://github.com/clojure-emacs/cider-nrepl/pull/693

Reply:
Oh, sure, if that's acceptable, I'd be happy to.  I'd just assumed that the binary had been intentionally preferred, and I'd have to check, but might even be able to have a `lein kondo` alias that does all that if it were desirable.

Reply:
Changed.